src/test/scala/com/rxthings/di/ExampleOfTestingSpec.scala

```diff
@@ -19,6 +19,7 @@ class ExampleOfTestingSpec extends InjectSpec with Matchers {
 }
 
 class InjectedActor extends NopActor {
+  import context._
```

Comment:
So now the import of `di._` is not longer enough, we require a second import to inject?